pkg/entity/flag_snapshot.go

```diff
@@ -66,4 +69,21 @@ func SaveFlagSnapshot(db *gorm.DB, flagID uint, updatedBy string) {
 	if err := tx.Commit().Error; err != nil {
 		tx.Rollback()
 	}
+
+	logFlagSnapshotUpdate(flagID, updatedBy)
+}
+
+var logFlagSnapshotUpdate = func(flagID uint, updatedBy string) {
+	if config.Global.StatsdClient == nil {
+		return
+	}
+
+	config.Global.StatsdClient.Incr(
+		"flag.snapshot.updated",
+		[]string{
+			fmt.Sprintf("FlagID:%d", flagID),
+			fmt.Sprintf("UpdatedBy:%s", util.SafeStringWithDefault(updatedBy, "null")),
```

Comment:
what are the values range for updatedBy tag?
we should use relative bucket value (such as 1d-ago, 2d-ago), rather than absolute value?

Reply:
updatedBy is not a timestamp, it's the authenticated user